core/controllers/learner_goals.py

```diff
@@ -57,6 +57,12 @@ class LearnerGoalsHandler(
         'DELETE': {}
     }
 
+    # Note: Currently this handler only accepting one type of activity
+    # which is 'ACTIVITY_TYPE_LEARN_TOPIC', so 'activity_type' argument
+    # is not used inside the function body but, in future if this handler
+    # accepts more than one activity of type then please remove pylint
```

Comment:
...one activity type, then please remove the pylint exception below and use an 'if-else' ...

Reply:
Done